apps/teams/models.py

```diff
@@ -180,7 +180,15 @@ def is_active_for_team(self, team):
         if not team:
             return False
 
-        if not self.pk:
+        if not self.pk and get_setting("CREATE_MISSING_FLAGS"):
+            flag, _created = Flag.objects.get_or_create(
+                name=self.name, defaults={"everyone": get_setting("FLAG_DEFAULT")}
+            )
+            self.id = flag.id
+            self.refresh_from_db()
+            cache = get_cache()
+            cache.set(self._cache_key(self.name), self)
+        else:
             # flag not created
             return False
```

Comment:
This needs to be adjusted so that it always returns `False` if it didn't exist:
```suggestion
        if not self.pk:
            if get_setting("CREATE_MISSING_FLAGS"):
                flag, _created = Flag.objects.get_or_create(
                    name=self.name, defaults={"everyone": get_setting("FLAG_DEFAULT")}
                )
                self.id = flag.id
                self.refresh_from_db()
                cache = get_cache()
                cache.set(self._cache_key(self.name), self)

            return get_setting('FLAG_DEFAULT')
```